sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -11,7 +11,40 @@ This proposal details what is required to provide this information as a SYCL ext
 
 The Feature Test Macro will be defined as:
 
-    #define SYCL_EXT_INTEL_DEVICE_INFO 1
+    #define SYCL_EXT_INTEL_DEVICE_INFO 2
```

Comment:
Why did you change macro value to 2?

Reply:
wanted to treat it as a versioning for the extensions added. any value >= 2 will have UUID support.

Reply:
I don't think it was designed this way, but don't actually see why we couldn't use it this way. @gmlueck , what do you think?

Reply:
As SYCL 2020 spec says (6.3.3 Feature test macros) this **is** the way it was designed:

> Since the interface to an extension might change from one release to another, vendors are also encouraged to predefine the macro’s value to the version of the extension. Vendors should use a numerical value that monotonically increases for each revision of the extension API.

Reply:
OK, great.